hazelcast/src/main/java/com/hazelcast/sql/SqlService.java

```diff
@@ -26,36 +26,27 @@
 /**
  * A service to execute SQL statements.
  * <p>
- * The service is in beta state. Behavior and API might change in future releases. Binary compatibility is not
- * guaranteed between minor or patch releases.
+ * If this cluster has Jet enabled, a statement can be executed by SQL backend as Jet job.
+ * The {@code hazelcast-sql} must be in the classpath, otherwise an exception will be thrown.
```

Comment:
```suggestion
 * In order to use the service, Jet engine must be enabled - SQL statements are executed as
 * Jet jobs. On members, the {@code hazelcast-sql.jar} must be on the member's classpaths,
 * otherwise an exception will be thrown; on client, it is not necessary.
```